CMakeLists.txt

```diff
@@ -23,7 +23,19 @@ target_sources(
     src/shared.c
 )
 
-if(APPLE)
+if(ANDROID)
+  target_sources(
+    log
+    PRIVATE
+      src/android.c
+  )
+
+  target_link_libraries(
```

Comment:
It was a solution for: https://stackoverflow.com/questions/4455941/undefined-reference-to-android-log-print
probably not the best approach